backend/protos/xyz/block/ftl/v1/language/language.proto

```diff
@@ -2,11 +2,132 @@ syntax = "proto3";
 
 package xyz.block.ftl.v1.language;
 
+import "google/protobuf/struct.proto";
+import "xyz/block/ftl/v1/ftl.proto";
 import "xyz/block/ftl/v1/schema/schema.proto";
 
 option go_package = "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/language;languagepb";
 option java_multiple_files = true;
 
+// ModuleConfig contains the configuration for a module, found in the module's ftl.toml file.
+message ModuleConfig {
+  // name of the module
+  string name = 1;
+  // absolute path to the module's directory
+  string path = 2;
+
+  // absolute path
+  string deployDir = 3;
+  optional string build = 4;
+  optional string generated_schema_dir = 5;
+  repeated string watch = 6;
+
+  // LanguageConfig contains any metadata specific to a specific language.
+  // These are stored in the ftl.toml file under the same key as the language (eg: "go", "java")
+  google.protobuf.Struct language_config = 7;
```

Comment:
Should we make this strongly typed using [`google.protobuf.Any`](https://protobuf.dev/programming-guides/proto3/#any)? We'd likely need some mechanism to pass the descriptor to FTL from the language plugin so it knows what to expect, so I'm not sure it's worth the hassle.

Reply:
Yeh I tried with `google.protobuf.Any` but we needed the proto definition for each language config, which was why I backtracked to just a generic struct. I didn't think it would be worth the effort for now.